docs/ossm/ossm2-migration/cluster-wide/README.md

```diff
@@ -0,0 +1,384 @@
+    details-v1-7f46897b-d497c.bookinfo            Kubernetes     SYNCED          SYNCED          SYNCED          SYNCED          NOT SENT     istiod-install-istio-system-866b57d668-6lpcr     1.20.8
+    productpage-v1-74bfbd4d65-vsxqm.bookinfo      Kubernetes     SYNCED (4s)     SYNCED (4s)     SYNCED (3s)     SYNCED (4s)     IGNORED      istiod-797bb4d78f-xpchx                          1.24.1
+    ratings-v1-559b64556-c5ppg.bookinfo           Kubernetes     SYNCED          SYNCED          SYNCED          SYNCED          NOT SENT     istiod-install-istio-system-866b57d668-6lpcr     1.20.8
+    reviews-v1-847fb7c54d-qxt5d.bookinfo          Kubernetes     SYNCED          SYNCED          SYNCED          SYNCED          NOT SENT     istiod-install-istio-system-866b57d668-6lpcr     1.20.8
+    reviews-v2-5c7ff5b77b-8jbhd.bookinfo          Kubernetes     SYNCED          SYNCED          SYNCED          SYNCED          NOT SENT     istiod-install-istio-system-866b57d668-6lpcr     1.20.8
+    reviews-v3-5c5d764c9b-rrx8w.bookinfo          Kubernetes     SYNCED          SYNCED          SYNCED          SYNCED          NOT SENT     istiod-install-istio-system-866b57d668-6lpcr     1.20.8
+    ```
+    In case you restarted all deployments, all proxies will be upgraded:
+    ```sh
+    $ istioctl ps -n bookinfo
+    NAME                                           CLUSTER        CDS              LDS              EDS             RDS              ECDS        ISTIOD                             VERSION
+    details-v1-7b5c68d756-9v9g4.bookinfo           Kubernetes     SYNCED (13s)     SYNCED (13s)     SYNCED (4s)     SYNCED (13s)     IGNORED     istiod-797bb4d78f-xpchx            1.24.1
+    productpage-v1-db9bfdbd4-z5c2l.bookinfo        Kubernetes     SYNCED (9s)      SYNCED (9s)      SYNCED (4s)     SYNCED (9s)      IGNORED     istiod-797bb4d78f-xpchx            1.24.1
+    ratings-v1-7684d8d8b8-xzrc6.bookinfo           Kubernetes     SYNCED (12s)     SYNCED (12s)     SYNCED (4s)     SYNCED (12s)     IGNORED     istiod-797bb4d78f-xpchx            1.24.1
+    reviews-v1-fb4d48bd8-lzvtx.bookinfo            Kubernetes     SYNCED (12s)     SYNCED (12s)     SYNCED (4s)     SYNCED (12s)     IGNORED     istiod-797bb4d78f-xpchx            1.24.1
+    reviews-v2-58bcc78ff6-fcrb8.bookinfo           Kubernetes     SYNCED (11s)     SYNCED (11s)     SYNCED (4s)     SYNCED (11s)     IGNORED     istiod-797bb4d78f-xpchx            1.24.1
+    reviews-v3-5d56c9c79b-l6gms.bookinfo           Kubernetes     SYNCED (11s)     SYNCED (11s)     SYNCED (4s)     SYNCED (11s)     IGNORED     istiod-797bb4d78f-xpchx            1.24.1
+    ```
+You can now proceed with the migration of next namespace.
+
+  > **_NOTE:_** Even with different versions of the proxies, the communication between services should work normally.
+
+#### Remove 2.6 control plane
```

Comment:
Is there any difference in removal between multi-tenant and cluster-wide? If not, perhaps this can be a separate page on its own (so as not to duplicate).

Reply:
There will be some minor differences but it should be possible to have some general steps which will work for both. I will remove the cleaning section from here and it will be done in https://issues.redhat.com/browse/OSSM-8511